core/templates/pages/exploration-editor-page/services/graph-data.service.spec.ts

```diff
@@ -88,7 +88,7 @@ describe('Graph Data Service', () => {
           },
           default_outcome: {
             dest: 'Hola',
-            dest_if_really_stuck: null,
+            dest_if_really_stuck: 'Hola',
```

Comment:
Why is the stuck state same as the default state?

Reply:
Actually, two states existed initially in the test. The purpose of the test is just to ensure that the right number and types of links are generated. So, it shouldn't really be a problem

Reply:
Ok, but just to be sure, we aren't assigning the stuck state to the normal state by default, right?

Reply:
No, we won't be doing that.